.github/workflows/update-snapshots.yml

```diff
@@ -31,7 +39,7 @@ jobs:
         with:
           repository: ${{ github.event.pull_request.head.repo.full_name }}
           ref: ${{ github.event.pull_request.head.ref }}
-          token: ${{ secrets.GIX_CREATE_PR_PAT }}
+          token: ${{ steps.app-token.outputs.token }}
```

Comment:
So I'm wondering if this is needed at all here. It looks like the `oisy-backend` action only pushes a docker image, it doesn't create a PR, so I think you could remove the token here and add permissions to the workflow.

Reply:
You are right! Removed